src/frontend/src/utils/getAccessToken.ts

```diff
@@ -0,0 +1,12 @@
+import { API_ROUTES } from '../constant/apiRoutes';
+
+export const getAccessToken = async (code: string | null) => {
+  if (!code) return;
+  const options = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' }, // by default setting the content-type to be json type
+    body: JSON.stringify({ code: code }),
+  };
+  const res = await fetch(`/${API_ROUTES.KEYCLOAK_CALLBACK}`, options);
```

Comment:
We do not need to repeat ourselves. There is already a `HttpRequest` util which can be leveraged for making these calls rather than a siloed fetch.

Reply:
i will change it